upup/pkg/fi/cloudup/awstasks/route.go

```diff
@@ -115,11 +122,14 @@ func (s *Route) CheckChanges(a, e, changes *Route) error {
 		if e.Instance != nil {
 			targetCount++
 		}
+		if e.NatGateway != nil {
+			targetCount++
+		}
 		if targetCount == 0 {
-			return fmt.Errorf("InternetGateway or Instance is required")
+			return fmt.Errorf("InternetGateway | Instance | NatGateway is required")
```

Comment:
yah this does not make sense to the ops guy in me.  n00b using this needs to be ELI5.